srcpkgs/hplip/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'hplip'
 pkgname=hplip
-version=3.22.6
-revision=2
+version=3.22.10
+revision=1
 build_style=gnu-configure
 pycompile_dirs="usr/share/hplip"
 # configure checks sys.version[:3] for Python versioning, so 3.10 becomes 3.1;
```

Comment:
is this still necessary?

Reply:
I'll check. :+1:

Reply:
The python versioning does no longer break things. :+1:

I've fixed it with the recent push.